starter-code/css/main.css

```diff
@@ -6,3 +6,206 @@ Green: #00B172
 White: #FFF
 
 */
+
+@import url('https://fonts.googleapis.com/css?family=Quicksand');
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+
+
+body {
+  line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+body {
+    font-family: 'Quicksand', sans-serif;
+    width: 100%;
+    height: 100%;
+}
+
+/*Header - Main */
+
+header {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    background-color: white;
+    align-self: center;
+    justify-content: center;
+}
+
+li {
+    display: inline;
+}
+
+.alinear-principio {
+    align-self: start;
+}
+
+.alinear-final {
+    align-self: end;
+}
+
+.alinear-centro-inicio {
+    justify-self: start;
+    align-self: end;
+}
+
+.alinear-centro-final {
+    justify-self: end;
+    align-self: center;
+}
+
+.image-header{
+    width: 200px;
+    height: 60px;
+    padding-left: 15px;
+    padding-top: 15px;
+}
+
+#section1 {
+    background-image: url("../images/landing.jpg");
+    background-size: 100% 100%;
+    height: 700px;
+    display: grid;
+    justify-content: center;
+    align-self: center;
+    color:white;
+}
+
+#section2{
+    display:grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 20% 80%;
+    grid-template-areas:
+    "title title title"
+    "div1 div2 div3"
+}
+#section2 h2{
+    grid-area: title;
+    align-self:center;
+    justify-self:center;
+
+}
+.section2-div1{
+    grid-area: div1;
+}
+.section2-div2{
+    grid-area: div2;
+}
+.section2-div3{
+    grid-area:div3;
+}
+#section2 img{
+    width: 80px;
+}
+#section2 div{
+    justify-self:center;
+    text-align: center;
+}
+
+#section2 h3{
+    padding-bottom:20px;
+    padding-top:10px;
+
+}
```

Comment:
Entre algunas rules habéis colocado un espacio y en entre otras no. Mejor ser consistente en todo el documento.